homeassistant/components/plex/media_browser.py

```diff
@@ -19,6 +29,15 @@
     "Recently Added": "recentlyAdded",
 }
 
+ITEM_TYPE_MEDIA_CLASS = {
+    "album": MEDIA_CLASS_ALBUM,
+    "artist": MEDIA_CLASS_ARTIST,
+    "playlist": MEDIA_CLASS_PLAYLIST,
+    "season": MEDIA_CLASS_SEASON,
+    "show": MEDIA_CLASS_TV_SHOW,
+    "video": MEDIA_CLASS_VIDEO,
+}
```

Comment:
@jjlawren are these all the media types for plex?

Reply:
Season and show are, yes. Video is probably a good fallback choice to have.

Reply:
Great, thanks!

Reply:
However that's not an exhaustive list. There's also track, episode, movie, and perhaps a couple others I'm not thinking of at the moment. Perhaps "clip", but I'm not sure how that's used in practice.

Reply:
Ok, thanks for checking back! :+1: 

I'll add all that you list here.

Should we catch a `KeyError`, log a debug log and skip that item if we encounter a missing type?

Reply:
I added track, episode and movie types.